101-hdinsight-linux-with-existing-linked-storage-account/README.md

```diff
@@ -6,6 +6,9 @@
 ![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-hdinsight-linux-with-existing-linked-storage-account/FairfaxLastTestDate.svg)
 ![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-hdinsight-linux-with-existing-linked-storage-account/FairfaxDeployment.svg)
 
+![sanitize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)
```

Comment:
fix buttons (See other samples if needed)

Reply:
Could you check again?